config/services.go

```diff
@@ -323,6 +323,21 @@ func GetVirtualSpaces() []string {
 	return getVspaceKeys(viper.GetStringMap("virtual_spaces"))
 }
 
+func GetVirtualSpace(spaceName string) (*base.VirtualSpace, error) {
+	if !IsVirtualSpace(spaceName) {
+		return nil, fmt.Errorf("%q is not a configured virtual space.", spaceName)
+	}
+
+	virtuals, err := getVirtualSpaces()
+	if err != nil {
+		return nil, err
+	}
+
+	sp := virtuals[spaceName]
```

Comment:
Don't you want to return an error if there aren't any virtual space named `spaceName`?

Reply:
This can't happen as it was already checked above with `isVirtualSpace()`